app/models/mars_manifest.rb

```diff
@@ -54,7 +54,7 @@ def raw_data
       # TODO: Stop bypassing SSL check.
 
       # sub out vertical tabs, they are expected in mars exports
-      @raw_data ||=  open(url, { ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE }).read.gsub(/\t/, "\n")
+      @raw_data ||=  open(url, { ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE }).read.gsub(/[\t\v]/, " ")
```

Comment:
This regex is looking for tab immediately followed by vertical tab. You could use
`.tr("\t\v", ' ')`
or
`.gsub(\t|\v, ' ')`

Reply:
they're in brackets - its searching for any tab or vt